x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_requests_table.test.tsx

```diff
@@ -66,6 +69,11 @@ describe('CrawlRequestsTable', () => {
       expect(tableContent).toContain('Full');
     });
 
+    it('renders a domains column', () => {
+      expect(tableContent).toContain('Domains');
+      // TODO How to test for the contents of this badge?
```

Comment:
What is going on here?

Reply:
I'm not sure how to test for the existence/contents of this badge, like we do other values.  Searching for `1` doesn't help much.  Open to suggestions!

Reply:
Ah gotcha. Maybe just put a data-attribute on it? Not going to hold up your PR for this though. I approved it.

Reply:
oh data attribute, duh.  I'll follow-up